src/hooks/useOnSetDefaultVariant.ts

```diff
@@ -38,11 +38,11 @@ function useOnSetDefaultVariant(productId: string, variant: Node) {
       }
     },
   });
-  const onSetDefaultVariant = (selectedVariant = null) => {
+  const onSetDefaultVariant = (selectedVariant: Node | null = null) => {
     productVariantSetDefault({
       variables: {
         productId,
-        variantId: variant ? variant.id : selectedVariant.id,
+        variantId: variant ? variant.id : (selectedVariant?.id as string),
```

Comment:
I'm wondering if we should not throw here if there will be no `variant.id` or `selectedVariant?.id` instead - I'm wondering what will happen in runtime or it will be edge case? 🤔